### PR TITLE
PTRENG-3570_Azure setprivateip

### PR DESCRIPTION
**Pull request by Logeshwarsn**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Title of the PR starts with installer/product name (e.g. `[ansible/artifactory]`)
- [ ] CHANGELOG.md updated
- [ ] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by danielmkn — CHANGES_REQUESTED** [review]

File: AzureResourceManager/Artifactory/MP_submission/createUiDefinition.json [hunk 1]
```diff
@@ -48,7 +48,83 @@
...
+            "visible": true
+          },
+          {
+            "name": "ip_address",
+            "type": "Microsoft.Compute.UserNameTextBox",
+            "label": "IP Address",
+            "toolTip": "IP address to deploy",
+            "osPlatform": "Linux",
+            "constraints": {
+              "required": true,
```
[the remark this answers is not in the document]
Comment (Logeshwarsn) [reply AzureResourceManager/Artifactory/MP_submission/createUiDefinition.json]: Field was removed

File: AzureResourceManager/Artifactory/MP_submission/createUiDefinition.json [hunk 2]
```diff
@@ -48,7 +48,83 @@
...
+              "regex": "^[a-z0-9A-Z]{1,30}$",
```
[the remark this answers is not in the document]
Comment (Logeshwarsn) [reply AzureResourceManager/Artifactory/MP_submission/createUiDefinition.json]: sure, I have removed those verification coz this field wont required

**Comment by danielmkn** [comment]
There are no changes in the `azuredeploy.parameters.json`. This file should provide the identical set of parameters with `createUiDefinition.json`

**Comment by danielmkn** [comment]
`azuredeploy.json` has a problem, see the error below. I've tested the template from the master branch with the same parameters and the deployment was successful.
Artifactory UI is not starting btw, so looks like we have another problem to solve as well.   

`{'code': 'InvalidTemplateDeployment', 'message': "The template deployment 'azuredeploy' is not valid according to the validation procedure. The tracking id is 'b09b9220-f079-46c9-beb3-2157a4c7f480'. See inner errors for details."}

Inner Errors:
{'code': 'InvalidResourceReference', 'message': 'Resource /subscriptions/<subscription_id>/resourceGroups/daniel-artifactory-test/providers/Microsoft.Network/loadBalancers/rthaos3qxlb/frontendIPConfigurations/rthaos3qxlbprimaryFepoolprimaryIpConfig referenced by resource /subscriptions/<subscription_id>/resourceGroups/daniel-artifactory-test/providers/Microsoft.Network/loadBalancers/rthaos3qxlb/loadBalancingRules/HTTPRulePrimary was not found. Please make sure that the referenced resource exists, and that both resources are in the same region.'}`

Also, UI definition wasn't accepting master and join keys generated with `openssl rand -hex 32`, which is the recommended way. I've fixed that and committed.

**Review by danielmkn — DISMISSED** [review]

**Pull request closed without merging**